scipy/stats/tests/test_morestats.py

```diff
@@ -2604,30 +2611,35 @@ def test_directionalmean_higher_dim(self):
         directionalmean = stats.directionalmean(full_array, axis=2)
         assert_allclose(expected, directionalmean)
 
-    def test_directionalmean_list_ndarray_input(self):
+    @pytest.mark.parametrize("test_func", [stats.directionalmean,
+                                           stats.directionalvar])
+    def test_directionalfuncs_list_ndarray_input(self, test_func):
         # test that list and numpy array inputs yield same results
         data = [[0.8660254, 0.5, 0.], [0.8660254, -0.5, 0]]
         data_array = np.asarray(data)
-        directional_mean = stats.directionalmean(data)
-        directional_mean_array = stats.directionalmean(data_array)
-        assert_allclose(directional_mean, directional_mean_array)
+        assert_allclose(test_func(data), test_func(data_array))
 
-    def test_directionalmean_1d_error(self):
+    @pytest.mark.parametrize("test_func", [stats.directionalmean,
+                                           stats.directionalvar])
+    def test_directionalfuncs_1d_error(self, test_func):
         # test that one-dimensional data raises ValueError
         data = np.ones((5, ))
         message = (r"samples must at least be two-dimensional. "
                    r"Instead samples has shape: (5,)")
         with pytest.raises(ValueError, match=re.escape(message)):
-            stats.directionalmean(data)
+            test_func(data)
 
-    def test_directionalmean_normalize(self):
+    def test_directionalfuncs_normalize(self):
```

Comment:
Perhaps this and the next test could be replace by a single test that starts with non-normalized data and checks that the results are the same whether the statistic is taken from the original data with `normalize=True` or the data is normalized first then the statistic is taken with `normalize=False`.  

```python3
data = np.array([[0.8660254, 0.5, 0.], [1.7320508, -1., 0.]])
res = test_func(data, normalize=True)
normalized_data = data / np.linalg.norm(data, axis=-1)
ref = test_func(data, normalize=False)
assert_allclose(res, ref)
```
Is there anything else being tested here?

Reply:
Addressed.